pages/common/asciinema.md

```diff
@@ -0,0 +1,35 @@
+# asciinema
+
+> Record and replay terminal sessions, and optionally share them on asciinema.org.
+
+- Associate the local install of `asciinema` with an asciinema.org account:
+
+`asciinema auth`
+
+- Make a new recording (to be uploaded or saved locally):
+
+`asciinema rec`
+
+- Make a new recording and save it to a local file:
+
+`asciinema rec {{path/to/file}}.cast`
+
+- Replay a terminal recording from a local file:
+
+`asciinema play {{path/to/file}}.cast`
+
+- Replay a terminal recording hosted on asciinema.org:
+
+`asciinema play https://asciinema.org/a/{{cast_id}}`
+
+- Make a new recording, limiting any idle time to at most 2.5 seconds:
+
+`asciinema rec -i 2.5`
+
+- Print the full output of a locally saved recording:
```

Comment:
So is this like printing a transcript?

Reply:
I suppose so. I haven't tried this one TBH :sweat_smile:. I imagine it will either print both the typed commands and their output, or just the command outputs. The former seems more useful, so I'd venture saying it's more likely (I can't test it ATM).

Reply:
@sbrl just confirmed: this indeed prints both the commands and the output, with formatting and everything. So it's basically an instantaneous version of the `play` subcommand.

Do you think the command description could be changed to make this clearer?

Reply:
Hrm, now that you mention it, I think this is clear enough :-)